src/integrations/storage.js

```diff
@@ -32,12 +34,38 @@ const addOrUpdateEntry = (entries, newEntry) => {
 const entryNotDeleted = (entry, deletedEntryIds) =>
   !deletedEntryIds.includes(entry.sys.id);
 
+const correctDates = (entry: Object) => {
+  if (!entry.fields) {
+    return entry;
+  }
+
+  const { startTime, endTime } = entry.fields;
+
+  if (
+    startTime !== undefined &&
+    endTime !== undefined &&
+    isBefore(endTime[locale], startTime[locale])
```

Comment:
Shouldn't this be the other way around? If the `startTime` is before `endTime` no need to switch and return earlier right?